core/platform/email/mailgun_email_services.py

```diff
@@ -121,7 +121,7 @@ def send_email_to_recipients(
         server = (
             ('https://api.mailgun.net/v3/%s/messages')
             % feconf.MAILGUN_DOMAIN_NAME)
-        encoded_url = urllib.parse.urlencode(data)
+        encoded_url = urllib.parse.urlencode(data).encode('ascii')
```

Comment:
Why ascii and not utf-8 (which I believe is more standard)?

Reply:
Because it is recommended by the documentation.

Reply:
Can you add a comment with a link?

Reply:
Done.

Reply:
Done.